app/app.py

```diff
@@ -15,6 +15,8 @@
 queue_dict = defaultdict(deque)
 connecting_channels = set()
 
+dictID = 952075555866558484#
```

Comment:
こんな感じに、環境変数に持たせると良い気がします。
```suggestion
dictID = os.environ['DISCTCH_ID']
```